File: scratch/fix_report.js
const fs = require('fs');
const path = 'app/admin/report/page.tsx';
const content = fs.readFileSync(path, 'utf8');
const lines = content.split('\n');

// Line 886 is index 885
if (lines[885].includes('</Card>')) {
    console.log('Found extra card at 886, removing...');
    lines.splice(885, 1);
    fs.writeFileSync(path, lines.join('\n'), 'utf8');
    console.log('Fixed!');
} else {
    console.log('Extra card not found at expected position.');
}

File: scratch/fix_report.py
import os

path = r'c:\Users\BUSINESS ASIASISTEM\Downloads\dnd-coffee\app\admin\report\page.tsx'
with open(path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

new_lines = []
skip_next = False
for i in range(len(lines)):
    if skip_next:
        skip_next = False
        continue
    
    # Target line 886 (index 885)
    if i == 885 and '</Card>' in lines[i]:
        print(f"Skipping line {i+1}: {lines[i].strip()}")
        continue
    
    new_lines.append(lines[i])

with open(path, 'w', encoding='utf-8') as f:
    f.writelines(new_lines)
